redisinsight/ui/src/components/navigation-menu/NavigationMenu.tsx

```diff
@@ -186,6 +195,24 @@ const NavigationMenu = () => {
     },
   ]
 
+  const privateRdiRoutes: INavigations[] = [
+    {
+      tooltipText: 'Pipeline Management',
+      pageName: PageNames.rdiPipelineManagement,
+      ariaLabel: 'Pipeline Management page button',
+      onClick: () => handleGoPage(Pages.rdiPipelinePrepare(connectedRdiInstanceId)),
+      dataTestId: 'pipeline-management-page-btn',
+      // TODO: get active by routes
```

Comment:
Is this a `TODO` we can resolve as a part of this card?

Reply:
No, it will be covered in the another story - need to change routing a bit and navigation